### PR TITLE
Add blockcommit bandwidth in bytes cases

### PR DESCRIPTION
**Pull request by chunfuwen**
blockcommit can specifiy bandwidth with bytes
positive and negative scenarios are covered

Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
cleaning libvirtd logs...
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.virsh.blockcommit.normal_test.single_chain.file_disk.local.no_ga.notimeout.base_as_middle.top_inactive



DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.virsh.blockcommit.error_test.illegal_blockcommit_bandwidth_value

File: libvirt/tests/src/virsh_cmd/domain/virsh_blockcommit.py [hunk 1]
```diff
@@ -497,6 +519,12 @@ def check_chain_backing_files(disk_src_file, expect_backing_file=False):
             check_chain_backing_files(blk_source_image, expect_backing_file)
             return
 
+        # Start one thread to check the bandwidth in output
+        if bandwidth and bandwidth_byte:
+            bandwidth += 'B'
+            pool = ThreadPool(processes=1)
+            async_result = pool.apply_async(check_bandwidth_thread, (libvirt.check_blockjob, vm_name, blk_target, bandwidth, test))
```
Comment (Yingshun) [inline libvirt/tests/src/virsh_cmd/domain/virsh_blockcommit.py:526]: Don't we need to check async_result?

File: libvirt/tests/cfg/virsh_cmd/domain/virsh_blockcommit.cfg [hunk 2]
```diff
@@ -131,3 +134,8 @@
                     base_option = "base"
                     middle_base = "yes"
                     with_active_commit = 'yes'
+                - illegal_blockcommit_bandwidth_value:
+                    status_error = "yes"
```
Comment (Yingshun) [inline libvirt/tests/cfg/virsh_cmd/domain/virsh_blockcommit.cfg:138]: Is this line covered by L118?

**Comment by Yingshun** [comment]
> cleaning libvirtd logs...
> DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
> DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
> DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
> PASS 1-type_specific.io-github-autotest-libvirt.virsh.blockcommit.normal_test.single_chain.file_disk.local.no_ga.notimeout.base_as_middle.top_inactive

@chunfuwen  Can you please paste value of bandwidth_byte and blockcommit_bandwidth from output log?

File: libvirt/tests/cfg/virsh_cmd/domain/virsh_blockcommit.cfg (hunk 2 above)
Comment (chunfuwen) [reply libvirt/tests/cfg/virsh_cmd/domain/virsh_blockcommit.cfg:138]: updated

File: libvirt/tests/src/virsh_cmd/domain/virsh_blockcommit.py (hunk 1 above)
Comment (chunfuwen) [reply libvirt/tests/src/virsh_cmd/domain/virsh_blockcommit.py:526]: no need, result checking will happen in check_bandwidth_thread

File: libvirt/tests/cfg/virsh_cmd/domain/virsh_blockcommit.cfg (hunk 2 above)
Comment (chunfuwen) [reply libvirt/tests/cfg/virsh_cmd/domain/virsh_blockcommit.cfg:138]: updated

**Comment by chunfuwen** [comment]
@Yingshun , may I ask why need bandwidth_byte and blockcommit_bandwidth value?

File: libvirt/tests/src/virsh_cmd/domain/virsh_blockcommit.py (hunk 1 above)
Comment (chunfuwen) [reply libvirt/tests/src/virsh_cmd/domain/virsh_blockcommit.py:526]: updated

**Comment by Yingshun** [comment]
> @Yingshun , may I ask why need bandwidth_byte and blockcommit_bandwidth value?

Sure, I just want to confirm that the values are correct during this run.

**Comment by chunfuwen** [comment]
Below is the log when blockcommit_bandwidth is configured to 50   

 base_option = base
    blockcommit_bandwidth = 50


Running virsh command: blockcommit avocado-vt-vm1 vda --wait --verbose --base /var/tmp/avocado__2r_xsve/RHEL-8.snap1 --bandwidth 50 --bytes --top /var/tmp/avocado__2r_xsve/RHEL-8.snap2
Running virsh command: blockjob avocado-vt-vm1 vda --info
Running '/bin/virsh blockcommit avocado-vt-vm1 vda --wait --verbose --base /var/tmp/avocado__2r_xsve/RHEL-8.snap1 --bandwidth 50 --bytes --top /var/tmp/avocado__2r_xsve/RHEL-8.snap2'
Running '/bin/virsh blockjob avocado-vt-vm1 vda --info'
Command /bin/virsh blockjob avocado-vt-vm1 vda --info running on a thread
[stdout]     Bandwidth limit: 50 bytes/s (50.000 B/s)
[stdout]
[stderr]
Command '/bin/virsh blockjob avocado-vt-vm1 vda --info' finished with 0 after 0.03119373321533203s
[stderr] Block Commit: [  0 %]
status: 0
stdout: Bandwidth limit: 50 bytes/s (50.000 B/s)
stderr: Block Commit: [  0 %]
Bandwidth is equal to 50
[stdout]
[stdout] Commit complete
[stderr]

**Pull request merged**